### PR TITLE
feat #CCOP-108: add customer review support in Storefront app

### PR DESCRIPTION
**Pull request by YanBrovkin**

### CODE REVIEW

File: VirtoCommerce.Storefront/appsettings.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
       "Url": "http://localhost/admin",
       // Use AppId and SecretKey for Platform API authentication (has higher priority than UserName/Password)
       "AppId": "27e0d789f12641049bd0e939185b4fd2",
-      "SecretKey": "34f0a3c12c9dbb59b63b5fece955b7b2b9a3b20f84370cba1524dd5c53503a2e2cb733536ecf7ea1e77319a47084a3a2c9d94d36069a432ecc73b72aeba6ea78",
+      "SecretKey": "151f833d88386894c0da03471e65f57a6d22f0c16b06b7bfc716adf9f6b84718115d25fbd894949999055827fa6bf2f2d2f0bb1558d1b2ec884b272c0f3d5fbe",
```
Comment (eMazeika) [inline VirtoCommerce.Storefront/appsettings.json]: never expose your secret keys. Use "Client secrets" from visual studio instead

File: swagger.CustomerReviewsModule.Web.json [hunk 2]
```diff
@@ -0,0 +1,267 @@
+{
```
Comment (eMazeika) [inline swagger.CustomerReviewsModule.Web.json]: redundant. Remove this file

**Review by eMazeika — CHANGES_REQUESTED** [review]

File: VirtoCommerce.Storefront/appsettings.json (hunk 1 above)
Comment (YanBrovkin) [reply VirtoCommerce.Storefront/appsettings.json]: fixed

File: swagger.CustomerReviewsModule.Web.json (hunk 2 above)
Comment (YanBrovkin) [reply swagger.CustomerReviewsModule.Web.json]: removed

**Pull request merged**